simbad/util/tests/test_pdb_util.py

```diff
@@ -78,6 +68,113 @@ def test_molecular_weight_3(self):
 
         self.assertAlmostEqual(np.round(data, 0), np.round(reference_data, 0))
 
+    def test_calculate_integration_box(self):
+        """Test case for PdbStructure.integration_box"""
+
+        input_model = os.path.join(EXAMPLE_DIR, "toxd", "toxd.pdb")
+        pdb_struct = PdbStructure.from_file(input_model)
+        data = pdb_struct.integration_box
+        reference_data = (48.306749999999994, 56.73474999999999, 48.589749999999995, 19.84575)
+
+        self.assertEqual(data, reference_data)
+
+    def test_nchains_1(self):
+        """Test case for PdbStructure.nchains"""
+        input_model = os.path.join(EXAMPLE_DIR, "toxd", "toxd.pdb")
+        pdb_struct = PdbStructure.from_file(input_model)
+        data = pdb_struct.nchains
+        reference_data = 1
+
+        self.assertEqual(data, reference_data)
+
+    def test_nchains_2(self):
+        """Test case for PdbStructure.nchains"""
+        input_model = os.path.join(EXAMPLE_DIR, "rnase", "rnase.pdb")
+        pdb_struct = PdbStructure.from_file(input_model)
+        data = pdb_struct.nchains
+        reference_data = 2
+
+        self.assertEqual(data, reference_data)
+
+    def test_nres_1(self):
+        """Test case for PdbStructure.nres"""
+        input_model = os.path.join(EXAMPLE_DIR, "toxd", "toxd.pdb")
+        pdb_struct = PdbStructure.from_file(input_model)
+        data = pdb_struct.nres
+        reference_data = 59
+
+        self.assertEqual(data, reference_data)
+
+    def test_nres_2(self):
+        """Test case for PdbStructure.nres"""
+        input_model = os.path.join(EXAMPLE_DIR, "rnase", "rnase.pdb")
+        pdb_struct = PdbStructure.from_file(input_model)
+        data = pdb_struct.nres
+        reference_data = 192
+
+        self.assertEqual(data, reference_data)
+
+    def test_keep_first_chain(self):
+        """Test case for PdbStructure.keep_first_chain_only"""
+        input_model = os.path.join(EXAMPLE_DIR, "rnase", "rnase.pdb")
+        pdb_struct = PdbStructure.from_file(input_model)
+        pdb_struct.keep_first_chain_only()
+        data = pdb_struct.nchains
+        reference_data = 1
+
+        self.assertEqual(data, reference_data)
```

Comment:
Here you only check that it keeps one chain, but that does not necessarily mean that the code kept the **first** chain. Maybe check the chain name as well?